app-office/texstudio/patches/texstudio-2.12.14.patchset

```diff
@@ -0,0 +1,22 @@
+From 6146ad878331346219158e5d7f1a35946d1ea65a Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Tue, 22 Jan 2019 09:33:30 +0100
+Subject: we don't support -rdynamic flag
+
+
+diff --git a/texstudio.pro b/texstudio.pro
+index da5de73..3f1bbd8 100644
+--- a/texstudio.pro
++++ b/texstudio.pro
+@@ -429,7 +429,7 @@ exists(./.git)  {
+ #QMAKE_CXXFLAGS_DEBUG += -Werror  -Wall -Wextra -Winit-self -Wmain -Wmissing-include-dirs -Wtrigraphs -Wunused -Wunknown-pragmas -Wundef -Wpointer-arith -Wtype-limits -Wwrite-strings -Wclobbered -Wempty-body -Wsign-compare -Waddress -Wlogical-op -Winline
+ QMAKE_CXXFLAGS_DEBUG += -Wall -Wextra -Winit-self -Wmissing-include-dirs -Wtrigraphs -Wunused -Wunknown-pragmas -Wundef -Wpointer-arith -Wwrite-strings -Wempty-body -Wsign-compare -Waddress -Winline
+ QMAKE_CXXFLAGS += -std=c++0x
+-!win32: QMAKE_LFLAGS += -rdynamic # option not supported by mingw
++!win32: QMAKE_LFLAGS += # option not supported by mingw

```

Comment:
I think you ca get away with `!win32:!haiku` here.

Reply:
Didn't check that back then :)